doc/programming/samples.rst

```diff
@@ -150,21 +150,58 @@ This code sample simulates the passive dynamics of a given model, renders it off
 values, and saves them into a raw data file that can then be converted into a movie file with tools such as ffmpeg. The
 rendering is simplified compared to :ref:`simulate.cc <saSimulate>` because there is no user interaction, visualization
 options or timing; instead we simply render with the default settings as fast as possible. The dimensions and number of
-multi-samples for the offscreen buffer are specified in the MuJoCo model, while the simulation duration, frames-per-
-second to be rendered (usually much less than the physics simulation rate), and output file name are specified as
-command-line arguments. For example, a 5 second animation at 60 frames per second is created with:
+multi-samples for the offscreen buffer are specified in the MuJoCo model with the
+visual/global/{`offwidth <https://mujoco.readthedocs.io/en/stable/XMLreference.html#visual-global-offwidth>`_,
+`offheight <https://mujoco.readthedocs.io/en/stable/XMLreference.html#visual-global-offheight>`_} and
+visual/quality/`offsamples <https://mujoco.readthedocs.io/en/stable/XMLreference.html#visual-quality-offsamples>`_
+attributes, while the simulation duration, frames-per-second to be rendered (usually much less than the physics
+simulation rate), and output file name are specified as command-line arguments.
 
 .. code-block:: Shell
 
-     render humanoid.xml 5 60 rgb.out
+   record modelfile duration fps rgbfile [depthoverlay]
```

Comment:
So sorry, `depthoverlay` is unreadable, let's either revert to `adddepth` or just `depth`, up to you :)

Reply:
Haha, no worries, I reverted to adddepth then.